### PR TITLE
Add some useful things from SCMBinderTest to the test JAR

### PR DESCRIPTION
**Pull request by jglick**
Forgotten in #11.

@reviewbybees

### CODE REVIEW

**Comment by abayer** [comment]
:bee:

File: src/test/java/org/jenkinsci/plugins/workflow/multibranch/SCMBinderTest.java [hunk 1]
```diff
@@ -102,7 +102,7 @@
         WorkflowMultiBranchProjectTest.showIndexing(mp);
     }
 
-    static void assertRevisionAction(WorkflowRun build) {
+    public static void assertRevisionAction(WorkflowRun build) {
```
Comment (oleg-nenashev) [inline src/test/java/org/jenkinsci/plugins/workflow/multibranch/SCMBinderTest.java]: Some Javadocs would be useful

File: src/test/java/org/jenkinsci/plugins/workflow/multibranch/SCMBinderTest.java [hunk 2]
```diff
@@ -256,8 +256,8 @@ static void assertRevisionAction(WorkflowRun build) {
         r.assertLogContains("subsequent content", b);
         r.assertLogContains("not trusting", b);
     }
-    static class WarySource extends GitSCMSource {
-        WarySource(String id, String remote, String credentialsId, String includes, String excludes, boolean ignoreOnPushNotifications) {
+    public static class WarySource extends GitSCMSource {
```
Comment (oleg-nenashev) [inline src/test/java/org/jenkinsci/plugins/workflow/multibranch/SCMBinderTest.java]: doh

**Comment by svanoort** [comment]
🐝

File: src/test/java/org/jenkinsci/plugins/workflow/multibranch/SCMBinderTest.java (hunk 1 above)
Comment (jglick) [inline src/test/java/org/jenkinsci/plugins/workflow/multibranch/SCMBinderTest.java]: Well, it is for tests only—if you know you need to use it, you can look at the source and find usages.

**Pull request merged**